app/views/catalog/_search_form.html.erb

```diff
@@ -16,7 +16,11 @@
     <% end %>
 
     <label for="q" class="sr-only"><%= t('blacklight.search.form.search.label') %></label>
-    <%= text_field_tag :q, params[:q], placeholder: t('blacklight.search.form.search.placeholder'), class: "search_q q form-control", id: "q", autofocus: should_autofocus_on_search_box?, data: { autocomplete_enabled: autocomplete_enabled?, autocomplete_path: blacklight.suggest_index_path }  %>
+    <% if App.rails_5_1? %>
+      <%= text_field_tag :q, params[:q], placeholder: t('blacklight.search.form.search.placeholder'), class: "search_q q form-control", id: "q", autofocus: should_autofocus_on_search_box?, data: { autocomplete_enabled: autocomplete_enabled?, autocomplete_path: blacklight.suggest_index_path }  %>
+    <% else %>
+      <%= text_field_tag :q, params[:q], placeholder: t('blacklight.search.form.search.placeholder'), class: "search_q q form-control", id: "q", autofocus: should_autofocus_on_search_box?, data: { autocomplete_enabled: autocomplete_enabled? }  %>
```

Comment:
Seems like we can remove the entire `data` argument here, found this commit

https://gitlab.galileo.usg.edu/DLG/meta/-/merge_requests/359/diffs?commit_id=1157838179ef6d22946a33dc1329ebfe52d6afa5

Reply:
Good find! but since I already merged this PR let's make another one.